textproc/stork/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo   1.0
+PortGroup           github  1.0
+
+github.setup        jameslittle230 stork v1.5.0
+github.tarball_from archive
+revision            0
+
+version             1.5.0
+categories          textproc www
+platforms           darwin
+license             Apache-2
+maintainers         @neirbowj \
```

Comment:
typically we add here also an (obfuscated) e-mail address, see the guide or other ports